packages/mobx-state-tree/package.json

```diff
@@ -55,7 +55,7 @@
     "tape": "^4.6.0",
     "tslib": "^1.7.1",
     "tslint": "^3.15.1",
-    "typescript": "^2.4.2"
+    "typescript": "^2.7.1"
```

Comment:
Its a yarn project,
You'll need to upgrade via yarn for the lock file to be updated

Reply:
(done in latest commit)